examples/tutorials/gmt-config.py

```diff
@@ -0,0 +1,68 @@
+"""
+Configuring PyGMT defaults
+==========================
+
+Default GMT parameters can be set globally or locally using :class:`pygmt.config`
+"""
+
+import pygmt
+
+########################################################################################
+# Configuring default GMT parameters
+# ----------------------------------
+#
+# The user can override default parameters either temporarily (locally) or permanently
+# (globally) using :meth:`pygmt.config`. The full list of default parameters that can be
+# changed can be at :gmt-docs:`gmt.conf.html`.
+#
+# We demonstrate the usage of :meth:`pygmt.config` by configuring a map plot.
+
+# Start with a basic figure
+fig = pygmt.Figure()
+fig.basemap(region=[115, 119.5, 4, 7.5], projection="M10c", frame=True)
+fig.coast(land="black", water="skyblue")
+
+fig.show()
+
+########################################################################################
+# Globally overriding defaults
+# ----------------------------
+#
+# The `MAP_FRAME_TYPE` parameter specifies the style of map frame to use, of which there
```

Comment:
It's reST syntax, so we need to use double backticks instead of one.
```suggestion
# The ``MAP_FRAME_TYPE`` parameter specifies the style of map frame to use, of which there
```